app/views/users/index.html.erb

```diff
@@ -1 +1,9 @@
-<h1>List of users </h1>
\ No newline at end of file
+<h1>List of users </h1>
+<% @users.each do |user| %>
+    <%= render 'shared/user', user:user%>
+<%end%>
+<div class="button_holder">
+<button>
+<%= link_to 'Create new post', new_user_post_path(@current_user)%>
+</button>
+</div>
```

Comment:
When your app starts, it renders an error in your root path. Please look into it 👍🏽
![Screenshot from 2023-05-13 10-37-46](https://github.com/Donmark2k/blog_app/assets/44624138/bb85a63f-91ed-4a4d-8fcc-a58f83504e27)

Reply:
https://www.loom.com/share/974abb95d2ed4518a832671cc062b20a

Reply:
The application is starting from my own end

Reply:
I am open to a zoom call or slack message, I am currently online now